src/lib/routes/client-api/feature.ts

```diff
@@ -1,7 +1,11 @@
 import memoizee from 'memoizee';
 import { Response } from 'express';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import hasSum from 'hash-sum';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I don't fully understand why we need no-extraneous-dependencies. Is this a leftover from the initial tests?

At the risk of breaking compilation:
```suggestion
import hasSum from 'hash-sum';
```